src/components/JobTable/Actions.js

```diff
@@ -17,8 +17,10 @@ const Actions = ({ id, configurable }) => (
                 ) : (
                     <ViewJobAction id={id} />
                 )}
-                {configurable && <RunJobAction id={id} />}
-                {configurable && <DeleteJobAction id={id} />}
+                {configurable && (
+                    <RunJobAction enabled={enabled} id={id} refetch={refetch} />
```

Comment:
Very minor: I think it'd be clearer here if the prop wasn't called `refetch` but `onComplete`.
Then this line would be more understandable: `<RunJobAction ... onComplete={refetch} />`

Reply:
Yeah I agree, fixed in [4fccf8b](https://github.com/dhis2/scheduler-app/pull/495/commits/4fccf8bb666d8f2432e707666923c4173c019a8b)